src/main/java/frc/robot/filters/DriveInput.java

```diff
@@ -0,0 +1,71 @@
+package frc.robot.filters;
+
+import java.util.Objects;
+
+/**
+ *
+ * 
+ * @author aheitkamp
+ */
+public class DriveInput {
+    private double forward;
+    private double right;
+    private double rotation;
+
+    private boolean overrideYawLock = false;
+    private boolean overrideAutoYaw = false;
+
+    /**
+     *
+     * 
+     * @param Forward the speed level forward for the drive subsystem
+     * @param Left the speed level right for the drive subsystem
+     * @param Right the speed level right for the drive subsystem
+     */
+    public DriveInput(double Forward, double Right, double Rotation) {
+        forward = Forward;
+        right = Right;
+        rotation = Rotation;
+    }
+
+    public double getForward() { return forward; }
+    public double getRight() { return right; }
+    public double getRotation() { return rotation; }
+
+    public void setForward(double Forward) { forward = Forward; }
+    public void setRight(double Right) { right = Right; }
+    public void setRotation(double Rotation) { rotation = Rotation; }
+
+    public double[] get() {
+        double[] output = new double[3];
+
+        output[0] = (forward);
+        output[1] = (right);
+        output[2] = (rotation);
+
+        return output;
+    }
+
+    public boolean getOverrideYawLock() { return overrideYawLock; }
```

Comment:
formatting: standard formatting standards do not use single lines like this. spend the two extra return characters